.github/workflows/python-package.yml

```
@@ -32,4 +32,8 @@ jobs:
         uv run ruff check
     - name: Run Tests with Pytest
       run: |
-        uv run pytest -q
+        uv run pytest --cov --cov-branch --cov-report=xml
+    - name: Upload coverage reports to Codecov
```

actionlint (1.7.9)

step must run script with "run" section or run action with "uses" section

(syntax-check)